sbt-dump-event-schema/src/sbt-test/sbt-dumpschema/simple/test

```diff
@@ -0,0 +1,2 @@
+> dumpEventSchema
```

Comment:
Hmmm what's the purpose of this file? 🤔

Reply:
Maybe rename to sth more meaningful

Reply:
Or is it some sbt-test convention?

Reply:
It's a file that contains information on how to run a test, and what results to expect.
This is not a convention, but a requirement, all according to specification.